docs/source/tools/traffic_vault_util.rst

```diff
@@ -69,4 +69,12 @@ Usage
 
 	.. warning:: Although this flag is optional, the utility will not work without it. It will try, but it will fail\ [1]_.
 
+.. option:: --skip_tls
+
+	An optional flag which, if given, specifies whether to utilize TLS certificate checks when establishing a connection. Defaults to false.
+
+.. option:: --server_name
```

Comment:
Is this option really necessary? I.e. if we pass a hostname using `--vault_ip` (as the help text seems to indicate is ok), requiring this `--server_name` option seems redundant.

Reply:
I was curious about this to. I ran into it when testing. When I set the TLS configuration value `InsecureSkipVerify` to `true` I received a connection error:

```[connectionManager] tls: either ServerName or InsecureSkipVerify must be specified in the tls.Config```

I believe it is trying to match the ServerName to information in Certificate. Should I pass the `vault_ip` into the TLS configuration for `ServerName`?

Reply:
Oh, huh, TIL! I guess this isn't necessary when using the tlsConfig within an http client, but the riak client needs it. Yeah, let's just pass the `vault_ip` into the TLS config for `ServerName`. That would keep it from being redundant.

Reply:
Done!